packages/@ember/-internals/routing/lib/system/route.ts

```diff
@@ -2752,16 +2803,7 @@ if (ROUTER_EVENTS) {
     },
   };
 
-  Route.reopen(ROUTER_EVENT_DEPRECATIONS, {
-    _paramsFor(routeName: string, params: {}) {
```

Comment:
I'm pretty sure it was a bug for this to be in a conditional reopen. We always depend on `_paramsFor`.